script/main.py

```diff
@@ -59,16 +59,16 @@ def add_field_time_until_birthday(dob_fields):
     field["dob"].update(dob_new_field_time_until_birthday)
   
 
-def remove_special_characters_from_phone_numbers(phone_fields):
-  for field in phone_fields:
-    phone = field['phone']       
-    clear_phone = re.sub(r'\(|\)|\-|\+|\s', '', phone)
-    field['phone'] = clear_phone
+def clear_phone_numbers(phone):
```

Comment:
Since it clears only 1 number, it would be better to name this function as singular `clear_phone_number`